src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -151,6 +151,105 @@ $floating-surface-box-shadow-dark:
+        flex-shrink: 0;
+        color: var(--g-color-text-complementary);
+
+        &_pinned {
+            color: var(--g-color-text-secondary);
+        }
+
+        &:hover {
+            color: var(--g-color-text-primary);
+        }
+
+        :global(.g-button__icon) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)